packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -52,75 +57,111 @@ function QuickTalk ({
   const [_expand, setExpand] = React.useState(expand)
   const badge = (showBadge && comments.length > 0) ? comments.length : false
 
-  const QTButton = (fixedPosition) ? FixedButton : Button
+  const QTButtonContainer = (fixedPosition) ? FixedButtonContainer : ButtonContainer
   const QTPanel = (fixedPosition) ? FixedBox : Box
 
   if (!_expand) {
     return (
-      <QTButton
-        a11yTitle={t('QuickTalk.aria.openButton', { count: comments.length })}
-        onClick={() => setExpand(true)}
-        icon={<Chat />}
-        badge={badge}
-      />
+      <QTButtonContainer
+        background={{ dark: 'dark-3', light: 'light-3' }}
+        elevation='small'
+      >
+        <Button
+          a11yTitle={t('QuickTalk.aria.openButton', { count: comments.length })}
+          onClick={() => setExpand(true)}
+          icon={<Chat />}
+          badge={badge}
+        />
+      </QTButtonContainer>
     )
   }
 
   return (
-    <QTPanel
-      a11yTitle={t('QuickTalk.aria.mainPanel')}
-      elevation='medium'
-      role='dialog'
-      background={{ dark: 'dark-3', light: 'light-3' }}
-    >
-      <Box
-        direction='row'
-        flex={false}
-        justify='between'
-        pad='small'
+    <Keyboard onEsc={() => setExpand(false)}>
+      <QTPanel
+        a11yTitle={t('QuickTalk.aria.mainPanel')}
+        elevation='medium'
+        role='dialog'
+        background={{ dark: 'dark-3', light: 'light-3' }}
       >
-        <Anchor
-          a11yTitle={t('QuickTalk.aria.goToTalk')}
-          label={t('QuickTalk.headerLink')}
-          href={subject.talkURL}
-          target='_blank'
-          icon={<Chat />}
-        />
-        <Button
-          a11yTitle={t('QuickTalk.aria.closeButton')}
-          autoFocus={true}
-          icon={<Close size='small' />}
-          onClick={() => setExpand(false)}
-          plain
-        />
-      </Box>
-      <Box overflow='auto'>
-        <Box flex={false} pad={{ bottom: 'small', left: 'small', right: 'small' }}>
-          <UnorderedList as='ul' flex={false}>
-            {comments.map(comment => {
-              const author = authors[comment.user_id]
-              const roles = authorRoles[comment.user_id]
+        <Box
+          direction='row'
+          flex={false}
+          justify='between'
+          pad='small'
+        >
+          <Anchor
+            a11yTitle={t('QuickTalk.aria.goToTalk')}
+            label={t('QuickTalk.headerLink')}
```

Comment:
This looks like a modal heading, saying 'Subject discussion', when the box opens. I'd recommend removing the link and making that the actual title bar for the popup. Having the title bar of the popup be a link to another page is very confusing UX, and not something I've seen before. I don't think volunteers will understand or expect that they can click this text in order to go to Talk.

Reply:
If you remove the link, and make this a heading, then autofocus for the close button also makes more sense.